### PR TITLE
Sqlite implementation initial code

### PR DESCRIPTION
**Pull request by shreyas-damle**

### CODE REVIEW

File: pebblo/app/storage/sqlite_db.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+from .database import Database
+
+logger = get_logger(__name__)
+
+
+class SQLiteClient(Database):
+    def __init__(self):
+        super().__init__()
+        # Create an engine that stores data in the local directory's my_database.db file.
+        self.engine = create_engine("sqlite:///pebblo.db", echo=True)
```
Comment (dristysrivastava) [inline pebblo/app/storage/sqlite_db.py:16]: same here

File: pyproject.toml [hunk 2]
```diff
@@ -117,6 +117,7 @@ dependencies = [
   "xhtml2pdf==0.2.15",
   "python-bidi==0.4.2",
   "python-dateutil==2.9.0",
```
Comment (dristysrivastava) [inline pyproject.toml:119]: Don't we need to install `sqlite` here

File: pebblo/app/models/sqltable.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import JSON, Column, Integer, create_engine
+from sqlalchemy.ext.declarative import declarative_base
+
+# Create an engine that stores data in the local directory's my_database.db file.
+engine = create_engine("sqlite:///pebblo.db", echo=True)
```
[the remark this answers is not in the document]
Comment (shreyas-damle) [reply pebblo/app/models/sqltable.py:5]: I'll take care of this in the follow up PR. Also, we need to move it to cache dir.

File: pebblo/app/storage/sqlite_db.py (hunk 1 above)
Comment (shreyas-damle) [reply pebblo/app/storage/sqlite_db.py:16]: I'll take care of this in the follow up PR.

File: pyproject.toml (hunk 2 above)
Comment (shreyas-damle) [reply pyproject.toml:119]: I'll take care of this in the follow up PR.

**Pull request merged**